### PR TITLE
Add telemetry for azure maven plugin

### PR DESCRIPTION
**Pull request by Flanker32**
Add telemetry for azure maven plugin

### CODE REVIEW

File: azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/Constant.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ */
+
+package com.microsoft.azure.plugin.login;
+
+public class Constant {
+    public static final String TELEMETRY_KEY_AUTH_METHOD = "authMethod";
+    public static final String TELEMETRY_KEY_IS_BROWSER_AVAILABLE = "isBrowserAvailable";
+    public static final String TELEMETRY_KEY_IS_LOGGED_IN = "isLoggedIn";
+
+    public static final String TELEMETRY_VALUE_AUTH_METHOD_OAUTH = "OAuth ";
```
Comment (andxu) [inline azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/Constant.java]: remove trailing space

File: azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/LoginMojo.java [hunk 2]
```diff
@@ -75,4 +84,12 @@ public void execute() throws MojoFailureException {
             throw new MojoFailureException(String.format("Fail to login due to error: %s.", e.getMessage()));
         }
     }
+
+    protected void trackLogin(final String loginMethod) {
```
Comment (andxu) [inline azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/LoginMojo.java]: traceLoginMethod?

File: azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/Constant.java (hunk 1 above)
Comment (Flanker32) [reply azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/Constant.java]: Fixed

File: azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/LoginMojo.java (hunk 2 above)
Comment (Flanker32) [reply azure-maven-plugin/src/main/java/com/microsoft/azure/plugin/login/LoginMojo.java]: Done

**Pull request merged**